exercises/concept/parsing-log-files/go.mod

```
@@ -0,0 +1,3 @@
+module parsing_log_files
```

```suggestion
module parsinglogfiles
```

It is not idiomatic to use underscores in package names, see [this blog post](https://go.dev/blog/package-names).